inc/datahandlers/post.php

```diff
@@ -261,25 +261,25 @@ function verify_message()
 			if(stripos($db->type, 'my') !== false)
 			{
 				$fields = $db->show_fields_from("posts");
-				$dblimit = reset(array_filter(array_map(function($field)
+				foreach($fields as $field)
```

Comment:
I'd be tempted to use a function for this, as currently this loop isn't short-circuited so it will loop for every field in the posts table.

Reply:
Better now?